content/blog/2019-05-14-nasapower.md

```diff
@@ -0,0 +1,625 @@
+    ##  7 -117.  36.7  1983     1     7     7 1983-01-07    3.2    17.2 
+    ##  8 -117.  36.7  1983     1     8     8 1983-01-08    1.96   18.3 
+    ##  9 -117.  36.7  1983     1     9     9 1983-01-09    0.7    14.0 
+    ## 10 -117.  36.7  1983     1    10    10 1983-01-10    1.3    17.6 
+    ## # … with 13,139 more rows
+
+In the metadata header you can see information about where the data
+comes from and what dates have been queried and returned as well as the
+elevation data.
+
+The `tibble` contains a few columns not in the original data, but that
+can make it easier to work within R. The original data only include YEAR
+and DOY. Looking at the data returned, there are LON = the queried
+longitude as double; LAT = the queried latitude as double; YEAR = the
+queried year as double; MM = the queried month as double, DD = the
+queried day as double, DOY = the day of year or Julian date as integer,
+YYYYMMDD = the full date as a date object and the requested parameters
+T2M\_MIN and T2M\_MAX each as double.
+
+#### Visualising the Data
+
+To visualise these data I will use ggplot2, but first I need to gather the data
+into long format using tidyr’s `gather()`.
+
+``` r
+library(tidyr)
+library(ggplot2)
+library(hrbrthemes)
+
+dv_long <- 
+  tidyr::gather(dv, key = "T2M",
+                value = "Degrees",
+                c("T2M_MIN", "T2M_MAX"))
+
+ggplot(dv_long, aes(x = YYYYMMDD, y = Degrees,
+                        colour = Degrees)) +
+  geom_line() +
+  xlab("Year") +
+  ggtitle(label = "Death Valley, CA Daily Max and Min Temperatures",
+          sub = "Source: NASA POWER") +
+  scale_colour_viridis_c() +
+  theme_ipsum()
+```
+
+![](/img/blog-images/2019-05-14-nasapower/graph_t-1.png)<!-- -->
+
+That is quite a swing in air temperatures from well over 40˚ C to well
+below 0˚ C throughout the year. I was going to put together a comparison
+with station data using [GSODR](https://ropensci.github.io/GSODR/) but
+instead found a good reason why you might want to use nasapower to get
+POWER data. When I checked for stations nearby this specified point,
+there were two in the GSOD database, 746190-99999 and 999999-53139.
+However, neither one of them offers weather data for this time period.
+The station, 746190-99999, only offers data from 1982 to 1992 and
+999999-53139 only provides data from 2004 to 2019. This nicely
+illustrates one of the advantages of the POWER data, it is available for
+any point on the globe. One of the weaknesses you might have noticed if
+you looked at the elevation data in the metadata header is that it shows
+the elevation of Death Valley is over 1000 m above sea level when we
+know that it is 86 m below sea level. This is due to the fact that the
+data are the average of a 0.5˚ by 0.5˚ area or roughly
+3000 km<sup>2</sup> as opposed to a single point that a station can
+provide.
+
+#### Crop Modelling From Space
+
+However, this can be advantageous as well. Crop yield modelling often
```

Comment:
Maybe write as "This can be advantageous as well, however." since it's more engaging as a paragraph start